flake.nix

```diff
@@ -20,6 +21,7 @@
         (import ./hercules-ci.nix)
         (import ./pre-commit.nix)
         (import ./docs/build.nix)
+        ./runtimes/rust/lbr-prelude/build.nix
```

Comment:
Wait, this works? If it works, the import statements above could also just be file literals? Please try pretty please!

Reply:
Tried it, we can remove the imports